inputs/templates/terra_workspaces/cohort_mode/workflow_configurations/GatherBatchEvidence.json.tmpl

```diff
@@ -61,5 +61,6 @@
   "GatherBatchEvidence.samples": "${this.samples.sample_id}",
   "GatherBatchEvidence.melt_vcfs": "${this.samples.melt_vcf}",
   "GatherBatchEvidence.wham_vcfs": "${this.samples.wham_vcf}",
-  "GatherBatchEvidence.gvcfs": "${this.samples.gvcf}"
+  "GatherBatchEvidence.gvcfs": "${this.samples.gvcf}",
+  "GatherBatchEvidence.gvcf_gcs_project_for_requester_pays" : "${workspace.google_project_id}"
```

Comment:
I like this idea, but this does mean that "broad-firecloud-dsde-methods" will show up in the Workspace Data table that will get cloned along with the workflows - previously I just entered it manually to avoid this. Should also update dashboard to reflect that users should enter their own project in Workspace Data (and direct them to where to find the project ID)

Reply:
Actually now that you point this out, this workspace variable is populated using `cloud_env.google_project_id`, which is always going to be different from the project used for a Cromwell run. I've added another variable `cloud_env.terra_billing_project_id` for workspaces.